src/Client.ts

```diff
@@ -354,7 +361,6 @@ export default class Client {
       this.armServiceClient = new ArmServiceClient(this.serviceHost, grpcOptions)
       this.baseServiceClient = new BaseServiceClient(this.serviceHost, grpcOptions)
       this.boardServiceClient = new BoardServiceClient(this.serviceHost, grpcOptions)
-      this.cameraServiceClient = new CameraServiceClient(this.serviceHost, grpcOptions)
```

Comment:
don't we still need a good way to ask the Client for a camera? Like `client.camera("name")`?

Reply:
Well we do that in go due to type limitations. In typescript you could technically pass enough type info to know you need to pull a camera clients. Also the wrappers are so light that I'm wondering if we prioritize the Dx here

Reply:
> In typescript you could technically pass enough type info to know you need to pull a camera clients.

I added [this generic service method](https://github.com/viamrobotics/viam-typescript-sdk/pull/6/commits/7d97641f68c30fb88dc9e26fbb12f026e4213b99) that is meant to be called by the component clients, but it could be called in the way you describing to cut down on some code. 

> Also the wrappers are so light that I'm wondering if we prioritize the Dx here

That's fair, the wrappers did turn out to be on the lighter side. I'm still leaning towards not having to import everything into client and making it so users only need to import what they need.